src/components/Input/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+interface InputProps extends React.InputHTMLAttributes<HTMLInputElement> {
+  label?: string;
+  type: string;
+}
+
+const Input = ({ label, type, ...props }: InputProps): JSX.Element => {
+  return (
+    <div className="input-container">
+      <label className="input-label">{label}</label>
+      <input className="input" type={type} {...props} />
+    </div>
```

Comment:
We [tend to avoid](https://nimblehq.co/compass/development/code-conventions/css/#convention-26) using HTML tags as classnames, and we also want to be more in line with our BEM conventions.

I would suggest something like the following

```suggestion
    <div className="form-field">
      <label className="form-field__label">{label}</label>
      <input className="form-field__input" type={type} {...props} />
    </div>
```

If you have a better naming idea, you can go with that 😄

Reply:
Fixed in https://github.com/suphanatjarukulgowit/react-nimble-survey/commit/8172c6c9c8c48e41c595245d1a91cef8b76650c7